client-src/src/pages/inq-form1/inq-form1.html

```diff
@@ -57,79 +57,93 @@
         <ion-col no-padding>
-          <ion-row>
+          <!-- <ion-row>
             <ion-col *ngIf="diffState">
               <ion-item>
                 <ion-label floating>Country</ion-label>
                 <ion-select formControlName="country">
-                  <ion-option *ngFor="let country of countries" [value]=country.key>{{country.value}}</ion-option>
+                  <ion-option *ngFor="let country of countries" [value]=country>{{country}}</ion-option>
                 </ion-select>
               </ion-item>
             </ion-col>
             <ion-col *ngIf="diffState">
               <ion-item>
                 <ion-label floating>State</ion-label>
                 <ion-select formControlName="state">
-                  <ion-option *ngFor="let state of states" [value]=state.key>{{state.value}}</ion-option>
+                  <ion-option *ngFor="let state of states" [value]=state>{{state}}</ion-option>
                 </ion-select>
               </ion-item>
             </ion-col>
-          </ion-row>
+          </ion-row> -->
           <ion-row>
-            <ion-col>
+            <!-- <ion-col>
               <ion-item>
                 <ion-label floating>City</ion-label>
                 <ion-select formControlName="city">
-                  <ion-option *ngFor="let city of cities" [value]=city.key>{{city.value}}</ion-option>
+                  <ion-option *ngFor="let city of cities" [value]=city>{{city}}</ion-option>
                 </ion-select>
               </ion-item>
+            </ion-col> -->
+            <ion-col col-3 class="pin-autocomplete">
+              <!-- <ion-item> -->
+                <ion-label>Enter Your Pincode</ion-label>
+                <ng2-completer formControlName="pin" (selected)="onPincodeSelected($event)" [datasource]="pinService" [minSearchLength]="3"></ng2-completer>
+              <!-- </ion-item> -->
             </ion-col>
-            <ion-col>
+            <!-- <ion-col>
               <ion-item>
-                <ion-label floating>Pincode</ion-label>
-                <ion-select formControlName="pin">
-                  <ion-option *ngFor="let pincode of pincodes" [value]=pincode.key>{{pincode.value}}</ion-option>
+                <ion-label floating>Area</ion-label>
+                <ion-select formControlName="area">
+                  <ion-option *ngFor="let area of areas" [value]=area>{{area}}</ion-option>
                 </ion-select>
               </ion-item>
-            </ion-col>
-            <ion-col>
+            </ion-col> -->
+          </ion-row>
+          <ion-row *ngIf="areas">
+            <ion-col col-3>
               <ion-item>
                 <ion-label floating>Area</ion-label>
                 <ion-select formControlName="area">
-                  <ion-option *ngFor="let area of areas" [value]=area.key>{{area.value}}</ion-option>
+                  <ion-option *ngFor="let area of areas" [value]=area.name>{{area.name | titlecase}}</ion-option>
```

Comment:
Sorted on name

Reply:
Order by was deprecated from Angular, sorting will have to be done on server.

Reply:
Addressed in commit 48cab92.
Sorting done on client.